widget/assets/socWidgetInput.json

```diff
@@ -1,82 +1,218 @@
 {
-    "title": "", 
+    "title": "",
     "timeRange": "createDate",
     "days": 30,
-    "resource": "alerts", 
+    "resource": "alerts",
     "relatedResource": "incidents",
-    "relatedResource2":"assets",
-    "relatedResource3":"indicators",
-    "alertSource":{
-        "title":"Top 3 Data Sources"
+    "relatedResource2": "assets",
+    "relatedResource3": "indicators",
+    "alertSource": {
+        "title": "Top 3 Detections"
     },
-    "top3PlaybookRun":{
-        "title":"Top 3 Playbooks"
+    "top3PlaybookRun": {
+        "title": "Top 3 Playbooks"
     },
-    "topAlerts":{
-        "title":"Top 3 Alert Types"
+    "topAlerts": {
+        "title": "Top 3 Alert Types"
     },
-    "alerts":{
-        "title":"Alerts"
+    "alerts": {
+        "title": "Alerts"
     },
-    "truePositive":{
-        "title":"True positive"
+    "truePositive": {
+        "title": "True positive"
     },
-    "falsePositive":{
-        "title":"False positive"
+    "falsePositive": {
+        "title": "False positive"
     },
-    "closed":{
-        "title":"Closed"
+    "closed": {
+        "title": "Closed"
     },
-    "topIncidents":{
-        "title":"Top 3 Incident Types"
+    "automatedResolved": {
+        "title": "by Automation"
```

Comment:
IMO title should be capitalise

Reply:
It was discussed with Abhishek. No need to capitalise "by"